pkg/proxy/ipvs/proxier.go

```diff
@@ -168,6 +168,13 @@ const sysctlForward = "net/ipv4/ip_forward"
 const sysctlArpIgnore = "net/ipv4/conf/all/arp_ignore"
 const sysctlArpAnnounce = "net/ipv4/conf/all/arp_announce"
 
+// ipvsServiceSchedulerAnnotation allows specifying ipvs scheduler name per service.
+// The valid values are: rr, wrr, lc, wlc, lblc, lblcr, sh, dh, sed, nq.
+const ipvsServiceSchedulerAnnotation = "kube-proxy.kubernetes.io/ipvs-scheduler"
+// ipvsServiceLocalEndpointWeightAnnotation allows specifying a weight for ipvs real server
+// that located in on the current node.
+const ipvsServiceLocalEndpointWeightAnnotation = "kube-proxy.kubernetes.io/ipvs-local-endpoint-weight"
```

Comment:
This is effectively just a toggle for preferring local endpoints since you can only adjust the weight for local endpoints? Seems like this will be replaced with service topology.